package.json

```diff
@@ -2,18 +2,21 @@
   "name": "jose",
   "version": "0.0.0",
   "description": "JSON Object Signing and Encryption",
-  "main": "src/index.js",
+  "main": "lib/index.js",
   "directories": {
     "test": "test"
   },
+  "files": [
+    "lib",
+    "dist"
+  ],
   "scripts": {
-    "babel": "babel src -d lib",
-    "build": "npm run clean && npm run babel",
+    "build-lib": "babel src -d lib",
+    "build": "npm run clean && npm run build-lib",
     "clean": "rm -rf dist/* && rm -rf lib/*",
-    "webpack": "webpack --progress --colors --optimize-minimize --optimize-occurrence-order --optimize-dedupe",
-    "dist": "npm run build && npm run webpack",
-    "postinstall": "npm run build",
```

Comment:
The important bit here is the removal of this `postinstall` trigger. And the addition of the `files:` section.